server/app/controllers/TaskSolver.scala

```diff
@@ -26,20 +29,28 @@ class TaskSolver @Inject()(executor: RuntimeSuiteExecutor, dao: Dao, val message
 
   val solutionForm = Form {
     mapping(
-      solution -> nonEmptyAndDiffer(from = solutionTemplateText)
+      solution -> nonEmptyText
```

Comment:
we dropping nonEmptyAndDiffer from, but it is still in the code unused, let's add TODO or something